src/Components/DataHistoryExplorer/DataHistoryExplorer.tsx

```diff
@@ -375,6 +396,9 @@ const DataHistoryExplorer: React.FC<IDataHistoryExplorerProps> = (props) => {
                 TelemetryEvents.Tools.DataHistoryExplorer.UserAction
                     .ChangeChartOption;
             sendDataHistoryExplorerUserTelemetry(telemetry.eventName, [
```

Comment:
which hook? Since sendDataHistoryExplorerUserTelemetry is in a util file and I want to have a unique id for each modal control is mounted, that is why I thought I should pass it along to be appended to the emitted events:
![image](https://user-images.githubusercontent.com/45217314/219463222-77e1c816-b726-4133-acb9-927dbd8db231.png)

maybe I am missing something.